gui/styles/_wizard.styl

```diff
@@ -8,142 +8,89 @@
 
 .on-step-address
   .step
-    transform translate(-100%, 0)
+    transform translate(-100vw, 0)
 
 .on-step-folder
   .step
-    transform translate(-200%, 0)
+    transform translate(-200vw, 0)
 
 .step
   position absolute
   left 0
   top 0
   display flex
-  flex-direction column
+  //flex-direction column
```

Comment:
For cozy-desktop-v2, we had a transition between the pages of the wizard: sliding from one page to the next page. In CSS, it means that all the wizard pages were on the html DOM and we were just translating them to make the current one positioned in the view port. It can probably be simplified in cozy-desktop-v3, as the wizard now use OAuth (external pages) that makes this effect impossible.